dbt_coverage/__init__.py

```diff
@@ -756,11 +784,13 @@ def compute(project_dir: Path = typer.Option('.', help="dbt project directory pa
             model_path_filter: Optional[List[str]] = typer.Option(None, help="The model_path "
                                                                              "string(s) to "
                                                                              "filter tables "
-                                                                             "on.")):
+                                                                             "on."),
+            cov_format: CoverageFormat = typer.Option(None, help="The output format to print, either "
```

Comment:
@vvvito do you think we could default to `CoverageFormat.STRING_TABLE` here? I guess that would make it a bit more straightforward in the help message to pick up what the default is 🙂